specs-go/config.go

```diff
@@ -469,3 +479,77 @@ type Syscall struct {
 	Action Action `json:"action"`
 	Args   []Arg  `json:"args,omitempty"`
 }
+
+// Windows contains platform specific configuration for Windows based containers.
+type Windows struct {
+	// Resources contains constraints for containers.
+	Resources *WindowsResources `json:"resources,omitempty"`
+	// Networking contains the network connection settings for containers.
+	Networking *Networking `json:"networking,omitempty"`
+	// FirstStart is an optimization on first boot of a container.
+	FirstStart bool `json:"first_start,omitempty"`
```

Comment:
This property is likely best delegated to a runtime flag.  See previous discussion on flags vs. config settings [here][1] and [here][2], but the threshold seems to be close to “can you tell that was set from inside the container?”.

[1]: https://github.com/opencontainers/runtime-spec/issues/494#issuecomment-225251219
[2]: https://github.com/opencontainers/runtime-spec/pull/405#issuecomment-223488103